src/index.html

```diff
@@ -1,14 +1,73 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item ">
+            <a class="is-active nav__link" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item padding">
```

Comment:
```suggestion
          <li class="nav__item">
```

u do not need this extra class, use nav__item